src/_utils/item/index.tsx

```diff
@@ -1,171 +1,83 @@
-  leftAddon,
-  rightTitle,
-  rightTitleDisplay = TextDisplayType.TITLE,
-  rightTitleColor,
-  rightBody,
-  rightBodyDisplay = TextDisplayType.BODY,
-  rightBodyColor,
-  rightAddon,
-  tag = <div />,
-}: ItemProps) => {
-  let Tag = tag.type
-  let tagProps = tag.props
-  if (href) {
-    if (typeof href !== 'string') {
-      Tag = href.type
-      tagProps = { ...tagProps, ...href.props }
-    } else {
-      Tag = 'a'
-      tagProps = { href }
-    }
+  &:disabled {
+    cursor: default;
   }
-  const hasRightText = rightTitle || rightBody
 
-  return (
-    <Tag
-      {...tagProps}
-      onClick={onClick}
-      onFocus={onFocus}
-      onBlur={onBlur}
-      onMouseDown={onMouseDown}
-      className={cc([
-        'kirk-item',
-        {
-          'kirk-item--highlighted': highlighted,
-          'kirk-item--clickable': isClickable,
-        },
-        className,
-      ])}
-    >
-      {leftAddon && <div className="kirk-item-leftAddon">{leftAddon}</div>}
-      <div className="kirk-item-leftText">
-        {leftTitle && (
-          <Text
-            className={leftBody ? 'kirk-item-title' : null}
-            display={leftTitleDisplay}
-            textColor={leftTitleColor}
-            tag={TextTagType.DIV}
-          >
-            {leftTitle}
-          </Text>
-        )}
-        {leftBody && (
-          <Text
-            className="kirk-item-body"
-            display={leftBodyDisplay}
-            textColor={leftBodyColor}
-            tag={TextTagType.DIV}
-          >
-            {leftBody}
-          </Text>
-        )}
-        {leftBodyAnnotation && (
-            <Text
-                className="kirk-item-body-annotation"
-                display={leftBodyAnnotationDisplay}
-                textColor={leftBodyAnnotationColor}
-                tag={TextTagType.DIV}
-            >
-              {leftBodyAnnotation}
-            </Text>
-        )}
-      </div>
-      {hasRightText && (
-        <div className="kirk-item-rightText">
-          {rightTitle && (
-            <Text
-              className={rightBody ? 'kirk-item-title' : null}
-              display={rightTitleDisplay}
-              textColor={rightTitleColor}
-              tag={TextTagType.DIV}
-            >
-              {rightTitle}
-            </Text>
-          )}
-          {rightBody && (
-            <Text
-              className="kirk-item-body"
-              display={rightBodyDisplay}
-              textColor={rightBodyColor}
-              tag={TextTagType.DIV}
-            >
-              {rightBody}
-            </Text>
-          )}
-        </div>
-      )}
-      {rightAddon && <div className="kirk-item-rightAddon">{rightAddon}</div>}
-      {chevron && (
-        <div className="kirk-item-rightAddon">
-          {<ChevronIcon iconColor={!isClickable ? color.fadedText : color.secondaryText} />}
-        </div>
-      )}
-      <style jsx>{style}</style>
-    </Tag>
-  )
-}
+  /* Text areas */
+  .kirk-item-leftText {
```

Comment:
no `&` prefix ?

Reply:
Fixed :)

(they aren't necessary for children, but it's more consistent to have it everywhere)